### PR TITLE
Option for asm stringification. Implements #31. Implements #32

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by JBaczuk — CHANGES_REQUESTED** [review]
Very nice, I think this makes this module really flexible. It would be great to have these documented in the README. Just a couple minor things.

File: index.js [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+  const {
+    encoding = 'hex'
+  } = options
+
+  return BScript.fromRaw(rawScript, encoding).toAsm(options)
 }
 
-BScript.asmToRaw = function (asmScript, outputEncoding = null) {
-  return BScript.fromAsm(asmScript).toRaw(outputEncoding)
+BScript.asmToRaw = function (asmScript, optionsOrEncoding = null) {
```
[the remark this answers is not in the document]
Comment (wardlem) [reply index.js]: Good catch!

This has been fixed.

File: src/parseAsm.js [hunk 2]
```diff
@@ -108,7 +116,11 @@ function parseDataLiteral (chars, idx, terminator, start = idx) {
   return [literal(data, start), idx]
 }
 
-function parseTerm (chars, idx) {
+function parseTerm (chars, idx, options) {
```
[the remark this answers is not in the document]
Comment (wardlem) [reply src/parseAsm.js]: @JBaczuk No, I don't think so.  `parseTerm` is an internal function and it is always called with an options object.

**Comment by wardlem** [comment]
> It would be great to have these documented in the README.

I think that I would like to document everything for this release at the same time once everything else is finalized.

**Review by JBaczuk — APPROVED** [review]

**Pull request merged**